insights/specs/default.py

```diff
@@ -538,7 +538,8 @@ def block(broker):
     sysconfig_virt_who = simple_file("/etc/sysconfig/virt-who")
     sysctl = simple_command("/sbin/sysctl -a")
     sysctl_conf = simple_file("/etc/sysctl.conf")
-    sysctl_conf_initramfs = simple_command("/bin/lsinitrd /boot/initramfs-*kdump.img -f /etc/sysctl.conf /etc/sysctl.d/*.conf")
+    sysctl_conf_initramfs = simple_command(
+        "/bin/lsinitrd /boot/initramfs-*kdump.img -f /etc/sysctl.conf /etc/sysctl.d/*.conf")
```

Comment:
Did you mean to change this line?

Reply:
Looks like a formatting-only change to me.

Reply:
I did not, will revert.